server.js

```diff
@@ -65,8 +65,41 @@ app.prepare()
       cachedRender(req, res, '/docs/api')
     })
 
-    server.get('/sw.js', (req, res) => {
```

Comment:
This was accidentally deleted I suppose :)

Reply:
Ohhhhh, sorted